ansible/roles/vm_set/tasks/ptf_portchannel.yml

```diff
@@ -22,9 +22,9 @@
   set_fact:
     portchannel_config: "{{ topology['DUT']['portchannel_config'] | default({})}}"
 
-- name: Start PTF portchannel
+- name: Control PTF portchannel
   ptf_portchannel:
-    cmd: "start"
+    cmd: "{{ptf_portchannel_action}}"
```

Comment:
The common practice is to have a whitespace between the variable name for better readability, like `{{ ptf_portchannel_action }}`.

Reply:
Got it.